### PR TITLE
Select Multiple APK at once (Issue: #1439)

### PR DESCRIPTION
**Pull request by KISHORE-KUMAR-S**
Following Issue: #1439

I've written a simple logic to add multiple applications.

WhatsApp Image 2024-10-08 at 19 02 37 (2)
WhatsApp Image 2024-10-08 at 19 02 37 (1)
WhatsApp Image 2024-10-08 at 19 02 37

### CODE REVIEW

**Comment by Tienisto** [comment]
Thanks for the PR! Can you make it optional? Like via a toggle in the menu bar options?

**Comment by KISHORE-KUMAR-S** [comment]
> Thanks for the PR! Can you make it optional? Like via a toggle in the menu bar options?

Okay. I'll see through it.

**Comment by KISHORE-KUMAR-S** [comment]
A toggle switch has been added, allowing users to select either a single app or multiple apps.

WhatsApp Image 2024-10-08 at 21 48 30_88dcc87e
WhatsApp Image 2024-10-08 at 21 48 30_b0f828b3

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "cmake.sourceDirectory": "D:/Flutter Projects/localsend/app/windows"
```
Comment (Tienisto) [inline .vscode/settings.json:2]: Please remove this file.

File: app/devtools_options.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+description: This file stores settings for Dart & Flutter DevTools.
+documentation: https://docs.flutter.dev/tools/devtools/extensions#configure-extension-enablement-states
```
Comment (Tienisto) [inline app/devtools_options.yaml:2]: Please remove this file.

File: app/lib/gen/strings_en.g.dart [hunk 3]
```diff
@@ -8,1037 +8,1130 @@ part of 'strings.g.dart';
...
-	late final _StringsAboutPageEn aboutPage = _StringsAboutPageEn._(_root);
-	late final _StringsDonationPageEn donationPage = _StringsDonationPageEn._(_root);
-	late final _StringsChangelogPageEn changelogPage = _StringsChangelogPageEn._(_root);
-	late final _StringsAliasGeneratorEn aliasGenerator = _StringsAliasGeneratorEn._(_root);
-	late final _StringsDialogsEn dialogs = _StringsDialogsEn._(_root);
-	late final _StringsSanitizationEn sanitization = _StringsSanitizationEn._(_root);
-	late final _StringsTrayEn tray = _StringsTrayEn._(_root);
-	late final _StringsWebEn web = _StringsWebEn._(_root);
-	late final _StringsAssetPickerEn assetPicker = _StringsAssetPickerEn._(_root);
+  /// Returns the current translations of the given [context].
```
Comment (Tienisto) [inline app/lib/gen/strings_en.g.dart:11]: Please revert this file

File: .vscode/settings.json (hunk 1 above)
Comment (KISHORE-KUMAR-S) [reply .vscode/settings.json:2]: Okay

**Comment by KISHORE-KUMAR-S** [comment]
The file has been removed.

**Comment by KISHORE-KUMAR-S** [comment]
The above files has been removed and reverted as said.

**Comment by Tienisto** [comment]
Nice!

**Pull request merged**